Firestore/Example/FuzzTests/FSTFuzzTestsPrincipal.mm

```diff
@@ -47,6 +54,10 @@ int RunFuzzTestingMain() {
   char **argv = program_args;
   int argc = sizeof(program_args) / sizeof(program_args[0]);
 
+  // Initialize static objects.
+  DatabaseId database_id{"project", DatabaseId::kDefault};
+  serializer = new Serializer(database_id);
```

Comment:
It doesn't look like `serializer` has to be created dynamically in this case, so I suggest you avoid that (unless I'm missing something, in which case, carry on). This would automatically remove any questions about valid-vs-invalid pointer, who is responsible for deleting the object, when is the object deleted, etc.